apps/prairielearn/src/lib/user.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Parses a string of UIDs separated by commas, whitespace, line breaks, or semicolons
+ * into an array of unique UIDs.
+ */
+export function parseUidsString(uidsString: string): string[] {
+  const uids = new Set(
+    uidsString
+      .split(/[\s,;]+/)
+      .map((uid) => uid.trim())
+      .filter((uid) => uid),
+  );
+  return Array.from(uids);
```

Comment:
Since it's supposed to be unique anyway, any reason why you're not returning a set?

Reply:
🤷 most/all callsites I found were dealing with arrays, so this seemed natural. I suppose it's not too bad to rely on callers to do `Array.from(...)` themselves? I can change this if you'd like.

Reply:
I find returning a Set to be a bit unergonomic. I do like the deduplication of a Set, but I'd cast to a Set internally but then return an array.

Reply:
An `Iterable<string>` doesn't allow one to check its size without turning it back into an `Array` or `Set`. I'm inclined to move back to arrays completely.

Reply:
Done in 7d9eca8.